doc/Makefile.am

```diff
@@ -139,3 +139,6 @@ books-upload:
 www: clean-local deprecated-upload manhtml-upload global-upload doxygen-upload books-upload
 
 clean-local: global-clean manhtml-clean doxygen-clean abi-clean deprecated-clean
+
+check-recursive:
+	@true
```

Comment:
Add a comment why for these, also mentioning that we need to drop it if we add tests

Reply:
Fixed.